src/lib/containers/FileEntityHandleQueryWrapper.tsx

```diff
@@ -2,23 +2,26 @@ import React, { useEffect } from 'react'
 import {
   BatchFileRequest,
   FileEntity,
-  FileHandle,
+  ExternalFileHandle,
   FileHandleAssociateType,
   FileHandleAssociation,
   Row,
 } from '../utils/synapseTypes'
+import { S3FileHandle } from '../utils/synapseTypes/CloudProviderFileHandle'
 import {
   getEntityResult,
   getFiles,
 } from '../utils/SynapseClient'
 
+export type ExternalS3FileHandle = ExternalFileHandle | S3FileHandle
```

Comment:
The name `ExternalS3FileHandle` is confusing for type (sounds like an external S3 bucket).  Perhaps something more generic, like `FileHandle`?  Or `ExternalOrS3FileHandle`?

Reply:
Thanks, I have updated the name to `ExternalOrS3FileHandle`.